_pages/rsanti2.yaml

```
@@ -0,0 +1,5 @@
+---
+githubHandle: rsanti2
+pageUrl: rsanti2.github.io
+timestamp: 2018/10/20
```

The timestamp uses hyphens (-) and not slashes (/)
```suggestion
timestamp: 2018-10-20
```